src/App.tsx

```diff
@@ -1,61 +1,140 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { ComplitlyTodo, HendleEvent, TypeTodo } from './type';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const getUser = (todo: TypeTodo) => {
+  return usersFromServer.find(user => user.id === todo.userId);
+};
+
+const todos: ComplitlyTodo[] = todosFromServer.map(todo => ({
+  todo,
+  user: getUser(todo),
+}));
+
+const crateTodo = () => {
+  let curentId = 15;
+
+  const Todo = (title: string, userId: number): TypeTodo => {
+    curentId += 1;
+
+    return {
+      title,
+      userId,
+      id: curentId,
+      completed: false,
+    };
+  };
+
+  return Todo;
+};
+
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [stateTodos, setTodos] = useState(todos);
+  const [chooseUser, setUser] = useState('empty');
+  const [newTitle, setTitle] = useState('');
+  const [isErrorTitle, setErrorTitle] = useState(false);
+  const [isErrorSelect, setErrorSelect] = useState(false);
```

Comment:
```suggestion
  const [isErrorUser, setErrorUser] = useState(false);
```

isErrorSelect - плохое название, непонятно что происходит😥